builder/azure/arm/config.go

```diff
@@ -359,6 +359,13 @@ type Config struct {
 	// provisioning process.
 	CustomDataFile string `mapstructure:"custom_data_file" required:"false"`
 	customData     string
+	// Specify a file containing custom data to inject into the cloud-init
```

Comment:
`custom data` -> `user data` for consistency

Reply:
Good point!

Reply:
updated!